tox.ini

```diff
@@ -0,0 +1,25 @@
+[tox]
+minversion = 1.6
+skipsdist = True
+envlist = example,pep8
+
+[testenv]
+basepython = python2
+deps = -r{toxinidir}/conf/requirements.txt
+usedevelop = False
+commands =
+    jenkins-jobs --conf conf/jenkins_job.ini.example test reference/servers/{envname}:library -o {toxinidir}/output/{envname}
+
+[testenv:example]
```

Comment:
The [testenv:example] section is not needed. Once you mentioned `example` in the `envlist`, this env will be created and it will use default parameters from [testenv] section